oracle-plugin/src/e2e-test/java/io.cdap.plugin/OracleClient.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin;
+
+import io.cdap.e2e.utils.PluginPropertyUtils;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.TimeZone;
+
+/**
+ *  Oracle client.
+ */
+public class OracleClient {
+
+  private static Connection getOracleConnection() throws SQLException, ClassNotFoundException {
+    TimeZone timezone = TimeZone.getTimeZone("UTC");
+    TimeZone.setDefault(timezone);
+    Class.forName("oracle.jdbc.driver.OracleDriver");
+    String databaseName = PluginPropertyUtils.pluginProp("databaseName");
+    return DriverManager.getConnection("jdbc:oracle:thin:@//" + System.getenv("ORACLE_HOST")
+                                         + ":" + System.getenv("ORACLE_PORT") + "/" + databaseName,
+                                       System.getenv("ORACLE_USERNAME"), System.getenv("ORACLE_PASSWORD"));
+  }
+
+  public static int countRecord(String table, String schema) throws SQLException, ClassNotFoundException {
+    String countQuery = "SELECT COUNT(*) as total FROM " + schema + "." + table;
+    try (Connection connect = getOracleConnection(); Statement statement = connect.createStatement();
+         ResultSet rs = statement.executeQuery(countQuery)) {
+      int num = 0;
+      while (rs.next()) {
+        num = (rs.getInt(1));
+      }
```

Comment:
Why do we need a while loop here?
The count query should return only one row and that will be the count, isn't it??

Reply:
The `statement.executeQuery` returns an interface `ResultSet` which has a cursor initially pointed before its first row and needs to be moved forward using loop